tests/4.5/target/test_target_map_pointer_default.c

```diff
@@ -9,45 +9,51 @@
 // inside the omp target region. The array is then changed through the pointer.
 // Array is mapped as tofrom, while pointer is mapped with default value
 //
-////===----------------------------------------------------------------------===//
+////===------------------------------------------------------------------------===//
 
 #include <stdio.h>
 #include <omp.h>
+#include "ompvv.h"
 
 #define N 1000
 
-int main() {
-  int compute_array[N], *p;
-  int sum = 0, result = 0;
-  int i, isHost = -1;
+int test_default_tofrom() {
+  int compute_array[N];
+  int *p;	
+  int sum = 0, result = 0, errors = 0;
+  int i;
  
   for (i = 0; i < N; i++) 
     compute_array[i] = 0;
+  
   p = &compute_array[0];
 
-// p is mapped as a pointer to compute_array
+// p is default mapped tofrom as a pointer to compute_array
 #pragma omp target data map(tofrom: compute_array)
```

Comment:
Remove line 32 for default mapping, must keep this when testing implicit mapping